src/zcl_abapgit_repo_online.clas.abap

```diff
@@ -406,12 +411,18 @@ CLASS zcl_abapgit_repo_online IMPLEMENTATION.
 
     handle_stage_ignore( io_stage ).
 
+    IF get_sha1( ) IS INITIAL.
+      lv_parent = get_sha1_remote( ).
+    ELSE.
+      lv_parent = get_sha1( ).
```

Comment:
what is the difference between `get_sha1_remote` and `get_sha1`, and how can the developer tell which to use when?

Reply:
s. PR #4082